draft-ietf-oauth-attestation-based-client-auth.md

```diff
@@ -229,21 +233,23 @@ The following rules apply to validating the Client Attestation JWT. Application
 
 2. The JWT MUST contain an "exp" (expiration time) claim that limits the time window during which the JWT can be used.  The authorization server MUST reject any JWT with an expiration time that has passed, subject to allowable clock skew between systems.  Note that the authorization server may reject JWTs with an "exp" claim value that is unreasonably far in the future.
 
-3. The JWT MUST contain a "jti" (JWT ID) claim that provides a unique identifier for the token.  The authorization server MAY ensure that JWTs are not replayed by maintaining the set of used "jti" values for the length of time for which the JWT would be considered valid based on the applicable "exp" instant.
+3. The JWT MUST contain an "aud" (audience) claim containing a value that identifies the authorization server as an intended audience. The {{RFC8414}} issuer identifier URL of the authorization server MUST be used as a value for an "aud" element to identify the authorization server as the intended audience of the JWT.
```

Comment:
```suggestion
3. The JWT MUST contain an "aud" (audience) claim containing a value that identifies the authorization server as an intended audience. The {{RFC8414}} issuer identifier URL of the authorization server MUST be used as a value for an "aud" element to identify the authorization server as the intended audience of the JWT.
```
```suggestion
3. The JWT MUST contain the "aud" (audience) claim containing a value that identifies the authorization server as an intended audience. The {{RFC8414}} issuer identifier URL of the authorization server MUST be used as a value for an "aud" element to identify the authorization server as the intended audience of the JWT.
```

I suggest to have in the aud the url of the endpoint where the JWT is intended to be used, then the authz endpoint, the token endpoint ... it could/should be an array as well. When we say `containing a value that identifies the authorization server` we should mention the issuer (entity id) as well and a ref to rfc8414 (issuer claim, that MUST be present within the AS metadata)